x-pack/plugins/lens/public/shared_components/visual_options_popover/line_curve_option.test.tsx

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
I might miss something, but the visual options are only used for the xy visualization, right? In that case they should be located in the `xy_visualization` folder as well (`shared_components` is meant for things used in multiple chart types)